javascript/border.js

```diff
@@ -0,0 +1,135 @@
+    return activeDescription;
+  };
+  
+  var activeSoldStatus;
+
+  carLot.setActiveSoldStatus = function() {
+    activeSoldStatus = activeCard.firstChild.children[4];
+  };
+
+  carLot.getActiveSoldStatus = function() { 
+    return activeSoldStatus;
+  };
+
+  var activeBackgroundColor;
+  
+  
+  ///DEFAULT BORDER maker: runs at time of load within filloutCards to attach the default border color to each card. 
+  
+  carLot.setDefaultBorderColor = function(currentCarColor, carCard ) {
+
+    carCard.firstChild.setAttribute("style", `border-color: ${currentCarColor}`);
+
+  }; 
+
+
+
+  ///////////////////////
+  ///CARD CLICK FUNCTIONS
+  
+  //this is the first function that runs when something is clicked. Sets active card if you clicked on a div. 
+  carLot.whatWasClicked = function(event) {
+
+    
+    activeCard = event.currentTarget;
+
+    carLot.activateNavBar();
+    
+    carLot.setBorders(activeCard);
+    
+  };
+
+
+  //this is the second function that runs when a div is clicked. resets all active/inactive borders. 
+  carLot.setBorders = function(activeCard) {
+
+    var domCardArray = Array.from(document.getElementsByClassName("carCard"));
+    
+    domCardArray.forEach(function(card){
+
+      if (card.id === activeCard.id) {
+        carLot.setActiveCardInfo(activeCard);
+      } else {
+        carLot.resetBorder(card);
+      }
+
+    });
+
+  };
+
+
+  //this is the third function that runs when a div is clicked. Loops through inventory(json) array and sets active info variables. 
+  carLot.setActiveCardInfo = function( activeCard ) {
+
+    inventory = carLot.getInventory(); 
+   
+    for (var i = 0; i < inventory.length; i++) {
+      if (inventory[i].id === activeCard.id) {
+
+        associatedJsonInfo = inventory[i];
+
+        activeBackgroundColor = inventory[i].color;
+        carLot.setActiveSoldStatus(activeCard);
+        carLot.setActiveDescription(activeCard);
+        
+        break;
+      }
+    }
+
+    //now that I know the active background color, I'll set it into the CSS. 
+    carLot.setActiveBackgroundColor(activeCard, activeBackgroundColor);
+
+  };
+
+
+ 
+  //runs within .setborders each time active div is changed, to reset all non-active cards. 
+  carLot.resetBorder = function(cardThatWasNotClicked) {
+    cardThatWasNotClicked.firstChild.classList.remove("activeCard");
+     
+  }; 
+
+
+  //this is the function I'm most proud of in this project, although it's very tightly bound and I'm sure jquery will make it completely obselete. Drills into the project's stylesheet and resets the activeCard background color as the corresponding car color each time a new card is selected. Runs within .setBorders. 
+  carLot.setActiveBackgroundColor = function(activeCard, activeBackgroundColor) {
+    let stylesheet = document.styleSheets[1];
+    let activeCardClass = stylesheet.cssRules[2];
+
+    activeCardClass.style.backgroundColor = `${activeBackgroundColor}`;
```

Comment:
Since ```activeBackgroundColor``` is just a variable, and there's no other string-y stuff you're tacking onto it, the backticks, and ```${}``` are superfluous.  ````${activeBackgroundColor}```` and ```activeBackgroundColor``` are the same.